spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -436,6 +436,7 @@
 "gnu gpl": GPL-2.0-only
 "gpl (≥ 3)": GPL-3.0-or-later
 "http://ant-contrib.sourceforge.net/tasks/LICENSE.txt": Apache-1.1
+"http://asm.ow2.org/license.html": BSD-3-Clause
```

Comment:
Can you help me clarify my mis-understanding?

When does an entry go to simple-license-mapping, I though it was when there is no white space in the key, which is probably wrong assumption.

Reply:
My understanding was that simple-license-mapping is for one word licenses that are on the SPDX list everything else goes into declared-license-mapping

Reply:
"No whitespace" is not the *only* criteria. In general, everything that can be parsed (according to the Antlr syntax implemented according to the SPDX spec) as a single / simple license should go to `simple-license-mapping.yml`, and everything else should go to `declared-license-mapping.yml`. Also see the docs at

https://github.com/oss-review-toolkit/ort/blob/d4f0cb598cc9af4402be6c4f5ee599a6c8136bea/spdx-utils/src/main/kotlin/SpdxSimpleLicenseMapping.kt#L24-L29

and

https://github.com/oss-review-toolkit/ort/blob/d4f0cb598cc9af4402be6c4f5ee599a6c8136bea/spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt#L24-L28